javascript-typing-tutor/main.js

```diff
@@ -1,16 +1,15 @@
 var $phrase = document.getElementById('phrase-string');
-var string = 'whatever you are, be a good one.';
-var phraseArray = string.split('');
+var typingString = 'whatever you are, be a good one.';
 
 function appendPhrase(string) {
-  for (var i = 0; i < phraseArray.length; i++) {
+  for (var i = 0; i < typingString.length; i++) {
     var charSpan = document.createElement('span');
-    charSpan.textContent = phraseArray[i];
+    charSpan.textContent = typingString[i];
     $phrase.append(charSpan);
```

Comment:
Your function has a `string` parameter here that you can use instead of directly referring to `typingString`.